pkg/e2e/operators/operators.go

```diff
@@ -432,7 +432,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s roleBinding to exist", (timeoutDuration - elapsed), roleBindingName)
+				log.Printf("Waiting %v for %s roleBinding to exist", timeoutDuration-elapsed, roleBindingName)
```

Comment:
Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.